GCRCatalogs/alphaq.py

```diff
@@ -65,7 +66,9 @@ def _subclass_init(self, filename, **kwargs):
             'size_bulge_true':    'morphology/spheroidHalfLightRadius',
             'disk_sersic_index':  'morphology/diskSersicIndex',
             'bulge_sersic_index': 'morphology/spheroidSersicIndex',
-            'ellipticity_1':      'morphology/totalEllipticity1',
+            'position_angle':     (lambda pos_angle: pos_angle*(180.0/np.pi)**2,'morphology/positionAngle'),
```

Comment:
- Please confirm if `**2` in this formula is correct
- use `np.rad2deg` instead of `*(180.0/np.pi)`
- Please add one space after comma

Reply:
I incorrectly converted the units, so I have apply the conversion twice. The position angle comes out 0->180. I can apply the function twice...